scripts/tag-reconciler/tag-reconciler.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/cri-o/cri-o/internal/version"
+	"github.com/cri-o/cri-o/scripts/utils"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/release-sdk/git"
+	"sigs.k8s.io/release-utils/env"
+)
+
+func main() {
+	logrus.SetFormatter(&logrus.TextFormatter{DisableTimestamp: true})
+	if err := run(); err != nil {
+		logrus.Fatalf("Unable to %v", err)
+	}
+}
+
+const (
+	githubTokenEnvKey = utils.GithubTokenEnvKey
```

Comment:
You can probably use this variable directly. It's a constant already in the other package.